plugin.php

```diff
@@ -12,16 +12,119 @@
  * @package cache-collector
  */
 
-namespace Cache_Collector;
-
 if ( ! defined( 'ABSPATH' ) ) {
 	exit;
 }
 
+use Cache_Collector\Cache_Collector;
+
+require_once __DIR__ . '/src/class-cache-collector.php';
+
 /**
  * Instantiate the plugin.
  */
-function main() {
-	// ...
+function cache_collector_setup() {
+	Cache_Collector::register_post_type();
+
+	/**
+	 * Filter the threshold for cache key expiration.
```

Comment:
```suggestion
	 * Filters the threshold for cache key expiration.
```

Per https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#language.